data/appeals_and_other_reinstatements/by_calendar_year_half/trade_controls_compliance_by_calendar_year_half.csv

```diff
@@ -0,0 +1,17 @@
+year,half,government,type,count
```

Comment:
In most other files we also have a `government_iso2_code` column. Should we here too?

Reply:
❤️ added in https://github.com/github/transparency/pull/2/commits/de2d8cf38317684acc517be881a78d4d0e589d76